### ISSUE TITLE
LayoutReader: Evaluation metric ARD

### ISSUE THREAD

**Issue by WeihongM** [microsoft/unilm#577]
Hello, thanks for the impressive work.
I want to ask the calculation of the evaluation metric "ARD".
Do you calculate the ARD by dividing the number of test set? or just summing up all the ARD result on testset (without dividing the number of test set).
@HYPJUDY @zlwang-cs
Thanks.

**Comment by zlwang-cs** [comment]
Yes, we first calculate the "RD" for each page in the testing set and then divide it by the number of pages in the testing set so the final result is "ARD" aka "Average RD"

**Comment by hpc-liuyang** [comment]
@zlwang-cs @WeihongM Could you please let me know in which part of the code the 'ARD' is calculated? thanks
